buildpacks/nodejs-engine/tests/integration_test.rs

```diff
@@ -0,0 +1,59 @@
+#[cfg(test)]
+use libcnb_test::{BuildpackReference, IntegrationTest};
+use std::time::Duration;
+use ureq;
+
+#[test]
+fn test_node_with_indexjs() {
+    IntegrationTest::new(
+        "heroku/buildpacks:20",
+        "../../test/fixtures/node-with-indexjs",
+    )
+    .buildpacks(vec![BuildpackReference::Crate])
+    .run_test(|ctx| {
+        println!("{}", ctx.pack_stdout);
+        assert!(ctx
+            .pack_stdout
+            .contains("Detected Node.js version range: *"));
+        assert!(ctx.pack_stdout.contains("Installing Node.js"));
```

Comment:
`libcnb-test` now comes with `assert_contains!` which provides better UX for this case when the assertion fails (it prints the expected vs actual values rather than just an "expected true, got false".

See:
https://github.com/Malax/libcnb.rs/blob/424cdf94783cd2da10125c1edea540ddcfe6fdc0/examples/execd/tests/integration_test.rs#L21

(there are a few other usages below that could use it too)

Reply:
Addressed in [f0eb4d1](https://github.com/heroku/buildpacks-nodejs/pull/184/commits/f0eb4d1eeea404936e14272fffa03dd64825f44d).